homeassistant/components/alexa/flash_briefings.py

```diff
@@ -52,11 +55,24 @@ def get(self, request, briefing_id):
         """Handle Alexa Flash Briefing request."""
         _LOGGER.debug("Received Alexa flash briefing request for: %s", briefing_id)
 
-        if self.flash_briefings.get(briefing_id) is None:
+        if (
+            briefing_id == CONF_PASSWORD
```

Comment:
What is this check for ?

Reply:
Given the configuration example above, this is to ensure that requests to `http://localhost:8123/api/alexa/flash_briefings/YOUR_PASSWORD?password=YOUR_PASSWORD` will return 404. If I remove this check, things will fail later on where the code assumes that there is a dict behind this key.

See [this test](https://github.com/Tho85/home-assistant-core/blob/fix-alexa-flash-briefings/tests/components/alexa/test_flash_briefings.py#L98-L105).

Reply:
I've changed the code to make this more clear.